MTM-Backend/src/routes/v1/item/item.router.ts

```diff
@@ -0,0 +1,53 @@
+import express, {
+  type Request,
+  type Response,
+  type NextFunction,
+} from "express";
+import type { ItemInput, ItemInputNoID } from "../../../types/item";
+import * as ItemsService from "./item.service";
+import Joi from "joi";
+
+const itemsRouter = express.Router();
+
+/**
+ * gets an items based upon the {category, name}
+ */
+itemsRouter.get(
+  "/v1",
+  async (req: Request, res: Response, next: NextFunction) => {
+    const schema = Joi.object({
+      category: Joi.string(),
+      name: Joi.string(),
+    });
+    try {
+      const data = (await schema.validateAsync(req.body)) as ItemInput;
```

Comment:
We usually don't pass query params for get request in req.body. We just encode them as req.query. Look at https://github.com/ChangePlusPlusVandy/MotherToMother/pull/82/files for example